### PR TITLE
add SARSOP v0.96.0

### PR DESCRIPTION
**Pull request by MaximeBouton**
Moving from https://github.com/JuliaPOMDP/SARSOP_jll.jl to Yggdrasil 
JuliaPOMDP/POMDPs.jl#318

### CODE REVIEW

File: S/SARSOP/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    MacOS(:x86_64),
+    Windows(:i686),
+    Windows(:x86_64)
+]
+platforms = vcat(expand_cxxstring_abis.(platforms)...)
```
Comment (staticfloat) [inline S/SARSOP/build_tarballs.jl:38]:
```suggestion
platforms = expand_cxxstring_abis.(platforms)
```

File: S/SARSOP/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    MacOS(:x86_64),
+    Windows(:i686),
+    Windows(:x86_64)
+]
+platforms = expand_cxxstring_abis.(platforms)
```
Comment (giordano) [inline S/SARSOP/build_tarballs.jl:38]: Or without the dot?

File: S/SARSOP/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    MacOS(:x86_64),
+    Windows(:i686),
+    Windows(:x86_64)
```
Comment (giordano) [inline S/SARSOP/build_tarballs.jl:36]: Ok, let's do this for the time being, then we'll see what's going on
```suggestion
    Platform("i686", "linux"; libc="glibc"),
    Platform("x86_64", "linux"; libc="glibc"),
    Platform("x86_64", "macos"),
    Platform("i686", "windows"),
    Platform("x86_64", "windows"),
```

**Comment by giordano** [comment]
Registration failed with
```
25hERROR: LoadError: MethodError: no method matching LibGit2.GitHash(::Nothing)
Closest candidates are:
  LibGit2.GitHash(::NTuple{20, UInt8}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.6/LibGit2/src/types.jl:21
  LibGit2.GitHash() at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.6/LibGit2/src/types.jl:23
  LibGit2.GitHash(::LibGit2.GitHash) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.6/LibGit2/src/types.jl:24
  ...
Stacktrace:
 [1] init_jll_package(name::String, code_dir::String, deploy_repo::String; gh_auth::GitHub.OAuth2, gh_username::String)
   @ BinaryBuilder /depot/packages/BinaryBuilder/PtqUF/src/AutoBuild.jl:891
 [2] init_jll_package(name::String, code_dir::String, deploy_repo::String)
   @ BinaryBuilder /depot/packages/BinaryBuilder/PtqUF/src/AutoBuild.jl:856
 [3] top-level scope
   @ /agent/_work/1/s/.ci/register_package.jl:34
in expression starting at /agent/_work/1/s/.ci/register_package.jl:34
```
I'm going to kick it again to see if it gets resolved

**Comment by MaximeBouton** [comment]
Thanks for the fix on the platform, my initialize change was to resolve the method error.

**Comment by giordano** [comment]
After some troubles with GitHub, registration was successful: https://github.com/JuliaRegistries/General/pull/22279

**Comment by MaximeBouton** [comment]
Thank you very much @giordano @staticfloat !

**Pull request merged**